test/all-options.test.js

```diff
@@ -33,6 +27,14 @@ describe('when applied with all options', () => {
           return `License information can be found in ${licenseFile}`;
         },
       },
+      uglifyOptions: {
```

Comment:
hmm.. I basically agree 😛, but `uglifyOptions` is too verbose. What's your opinion on remvoing `warningFilters` && `extractComments` ? 🙃